tests/web/query/builder.py

```diff
@@ -63,6 +63,8 @@ def test_es():
     assert "_id" in query["_source"]["includes"]
     assert "fieldA" in query["_source"]["includes"]
 
+def test_regex_order
+
```

Comment:
is this unfinished?

Reply:
My bad I had moved the test to `tests/web/query/regex` and forgot to push it on Friday